src/Commands/DisplayErrors.php

```diff
@@ -36,13 +36,13 @@ class DebugMode extends Command
     protected function configure()
     {
         $this
-            ->setName('fop:debug')
-            ->setDescription('Configure debug mode')
-            ->setHelp('This command allows you to get or change debug mode')
+            ->setName('fop:errors')
+            ->setDescription('Configure and display _PS_MODE_DEV_')
```

Comment:
Should we use "debug mode" or mention internal _PS_MODE_DEV_ contant ? I prefer debug mode

Reply:
Yes, we can use 'debug mode' and add the information about _PS_MODE_DEV_ constant in the help text only.
Any other opinions ?

Reply:
_PS_MODE_DEV_ is now only in help text.